admin/data/sql/migrations/20190918140848.do.pupil-state.sql

```diff
@@ -0,0 +1,21 @@
+alter table mtc_admin.pupil
+	add currentCheckId int
+
+alter table mtc_admin.pupil
+	add checkComplete bit
+	constraint DF_checkComplete_Default default (0)
+
+alter table mtc_admin.pupil
+	add restartActive bit
+	constraint DF_restartActive_Default default (0)
+
+alter table mtc_admin.pupil
+	add attendanceId int
```

Comment:
Will we still be able to hold the history as teachers can remove the not attending flag?

Reply:
yes.  pupilAttendance will be retained and serve as a history table.  This simply gives us an optimal way to know whether to include the pupil in pin generation or not, and to see what the attendance reason is.

Reply:
That's great 😄